pyvelox/pyvelox.h

```diff
@@ -504,6 +519,48 @@ static void addVectorBindings(
             std::move(baseVector),
             PyVeloxContext::getSingletonInstance().pool());
       });
+
+  m.def(
+      "row_vector",
+      [](std::vector<std::string>& names,
+         std::vector<VectorPtr>& values,
+         std::function<bool(vector_size_t /*row*/)> isNullAt = nullptr) {
```

Comment:
Not sure that this is very useful to have at as a function (it makes sense when writing test cases).

Reply:
@kgpai would you rather suggest an input vector with flags for null in boolean and then use that to set the nulls?

Reply:
And it would resolve https://github.com/facebookincubator/velox/pull/6533#discussion_r1323668759 as well?

Reply:
Probably better to have an input vector, as we can use list comprehensions etc in python to generate it.

Reply:
Sure will do that.